SteamReview/SteamReview/Steam.cpp

```diff
@@ -148,21 +167,20 @@ void FSteam::AppRun()
 }
 
 //---------------------------------------------------------------
-int FSteam::ValidateInput(int InputValue, const int LowBoundValue, const int HighBoundValue)
+void FSteam::ValidateInput(int& InputValue, const int LowBoundValue, const int HighBoundValue)
 {
-
+	std::cin >> InputValue;
```

Comment:
Since you're assigning values to InputValue completely inside this function, maybe it would be better to return the input as a return type, rather than as an out parameter.

Ouput parameters are useful in some scenarios, like for example when you want to return a value but also return whether or not the returned value is valid, so you use a bool return type and an out parameter for the other type you want to return.

However, they are not the standard way of returning stuff, that's what the return type is for. With this in mind, although this method works fine, I believe that it would be clearer if you used the return type, the way you were doing before.

Ouput parameters should be special cases, rather than the rule. Especially with built-in types.